nighthawk/test/worker_test.cc

```diff
@@ -0,0 +1,61 @@
+#include <thread>
+
+#include "gtest/gtest.h"
+
+#include "common/api/api_impl.h"
+#include "common/filesystem/filesystem_impl.h"
+#include "common/runtime/runtime_impl.h"
+#include "common/stats/isolated_store_impl.h"
+
+#include "test/mocks/thread_local/mocks.h"
+
+#include "nighthawk/test/mocks.h"
+
+#include "nighthawk/source/common/worker_impl.h"
```

Comment:
We really need something like Envoy's check/fix_format scripts to enforce header ordering. Can you add this to the backlog if it doesn't already exist there?